src/plugins/intel_gpu/src/kernel_selector/kernels/resample/resample_kernel_pil_ref.cpp

```diff
@@ -119,6 +119,9 @@ DataTensor GetIntermediateBufferSize(const resample_params& params) {
     auto ybox_last = GetLastRow(params);
     std::vector<size_t> dims = output.LogicalDims();
     auto channelIndex = DataTensor::Channelndex(layout, Convert(params.axes[eVertical]));
+    if (channelIndex < 0) {
+        throw std::runtime_error("Invalid layout channel index");
+    }
```

Comment:
Please use `OPENVINO_ASSERT` instead of `if` + `throw`

Reply:
Applied it and thanks.